pkg/storage/blockdev.go

```diff
@@ -0,0 +1,391 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package storage
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/rekby/gpt"
+)
+
+var (
```

Comment:
to do: all this needs to be either // +build linux OR variables like this could go into a blockdev_linux.go file